Reference/Angular/index.md

```diff
@@ -0,0 +1,9 @@
+# Angular
+
+Since Umbraco 7, the Umbraco backoffice has been build using Angular. The implementation is made up of many directives, services and similar.
+
+Generally you can find information about these via the [Backoffice UI API documentation](/apidocs/v8/ui/). While this part of the documentation is auto-generated from the Umbraco source code, you can also find some more in-depth description and examples for these directives and services.
+
+## Directives
+
+- [**umbLoadIndicator**](Directives/umbLoadIndicator/)
```

Comment:
```suggestion
- [Load indicator](Directives/umbLoadIndicator/) (`<umbLoadIndicator />`)
```

Reply:
Yes, I did the same for the Directives page. Hadn't thought about that here 😊